pkg/cli/root.go

```diff
@@ -219,7 +219,7 @@ func validateConfiguration(config *config.Config) error {
 	if config.HTTP.RawAccessControl != nil {
 		if config.HTTP.Auth == nil || (config.HTTP.Auth.HTPasswd.Path == "" && config.HTTP.Auth.LDAP == nil) {
```

Comment:
I would move this `if` inside `checkForAnonymousPolicyConfig` / `validateAuthzPolicies ` (or whatever name it will finally have) as the relevant error message is there. Or move the error message from there over here, but I personally prefer the former.